app/cdap/styles/fonts/symbol-defs_new.svg

```diff
@@ -212,5 +212,17 @@
   <!-- <title>more</title> -->
   <path d="M12,8 C13.1,8 14,7.1 14,6 C14,4.9 13.1,4 12,4 C10.9,4 10,4.9 10,6 C10,7.1 10.9,8 12,8 L12,8 Z M12,10 C10.9,10 10,10.9 10,12 C10,13.1 10.9,14 12,14 C13.1,14 14,13.1 14,12 C14,10.9 13.1,10 12,10 L12,10 Z M12,16 C10.9,16 10,16.9 10,18 C10,19.1 10.9,20 12,20 C13.1,20 14,19.1 14,18 C14,16.9 13.1,16 12,16 L12,16 Z"></path>
  </symbol>
+<symbol id="icon-undo" width="18" height="18" viewBox="0 0 24 24">
```

Comment:
whats all this?

Reply:
They are the redo/undo icons I found from GM library, which were missing. But if we decided to use material ui icons, they can be removed